### DISCUSSION TITLE
Change already created Personal Presets in mod menu

### DISCUSSION THREAD

**Discussion by RosenbergGG** [ppy/osu#22827] [Ideas]
I'm not sure If this has already been brought up or I'm doing something wrong but editing already created presets by adding a new mod or removing one would be pretty neat. Right now I have DT 1.2 and wanted to add NF but I have to create a new one and customize the sliders again.

**Comment by Theighlin** [comment]
Fixed by #22801 if it gets merged.

**Reply by RosenbergGG** [reply]
Awesome, hope It does. Thanks!
